aries_cloudagent/DefaultMessageDelivery/OutboundMessageEmitter.py

```diff
@@ -0,0 +1,67 @@
+from aries_cloudagent.core.profile import Profile
+from aries_cloudagent.core.transport_events import (OutboundMessage,
+                                                    OutboundMessageEvent,
+                                                    OutboundStatusEvent)
+from aries_cloudagent.transport.inbound.manager import InboundTransportManager
+from aries_cloudagent.transport.inbound.message import InboundMessage
+from aries_cloudagent.transport.outbound.status import OutboundSendStatus
+
+from ..transport.outbound.queue.base import BaseOutboundQueue
+
+
+class OutboundMessageRouter:
+    """Outbound Message Router."""
+
+    def __init__(self):
+        pass
+
+    async def outbound_message_router(
+        self,
+        profile: Profile,
+        outbound: OutboundMessage,
+        inbound: InboundMessage = None,
+    ):
+        """Emit outbound message event."""
+        if (
+            not outbound.target
+            and outbound.reply_to_verkey
+            and not outbound.reply_from_verkey
+            and inbound
+        ):
+            outbound.reply_from_verkey = inbound.receipt.recipient_verkey
+
+        await profile.notify(event=OutboundMessageEvent(outbound))
```

Comment:
This should stay in conductor. This is not specific to the default delivery mechanism.

Reply:
I have moved that back into the conductor.